docs/Technical Reference.md

```diff
@@ -2,110 +2,102 @@ This page provides a technical reference on certain advanced configuration optio
 
 ## Deploying the T4 Catalog on AWS
 
-The following section uses AWS CloudFormation to deploy and manage the resources required to run a T4 bucket and catalog in your own VPC.
+The following instructions use CloudFormation to install T4 on a bucket in your AWS account.
 
+<!--
```

Comment:
Why is this commented out? If you don't want it in this doc, lift to another doc and link from here?

Reply:
I don't think point number 3 is important. Of course if users have administrative access they'll have access to everything, and of course if they have ElasticSearch access they'll have access to search results. That's a tautology.

Point number 1 isn't, in my mind, correct. The catalog is not single-bucket; you can now use it to browse any buckets that are configured with CORS and the other stuff.

Point number 2 is important, but a one-item list doesn't make sense.

And, from a organizational perspective, I don't think it makes sense to hit users with limitations in a document whose text is about installation. I think it belongs somewhere else.